Makefile

```diff
@@ -42,14 +42,17 @@ install:
 
 all: gen-project gendoc
 %.yaml: gen-project
-deploy: all mkd-gh-deploy
+deploy: gendoc mkd-gh-deploy
 
 # generates all project files
+# and updates the artifacts in linkml-model
 gen-project: $(PYMODEL)
 	$(RUN) gen-project -d $(DEST) $(SOURCE_SCHEMA_PATH) && mv $(DEST)/*.py $(PYMODEL)
+	cp -r $(DEST)/* $(PYMODEL)
+	rm -r $(PYMODEL)/docs
```

Comment:
Copy over all the latest artifacts from `target/docs`. Otherwise there is no point in generating all the artifacts in the first place?